src/views/password_reset/__tests__/PasswordResetConfirmContainer.test.js

```diff
@@ -52,4 +53,19 @@ describe('Tests PasswordResetConfirmContainer', () => {
     });
     expect(PasswordChangeBtn.length).toEqual(1);
   });
+  it('Test keyUp event', () => {
+    const form = wrapper.find(Form);
+    const inputField = form.find('FormControl[name="password_confirm"]');
+    const message = form.find('[className="matching"]');
+    // const input = {
+    //   password_confirm:form.find('input[name="password_confirm"]')
+    // };
+    Object.values(inputField).forEach(input => {
+      input.simulate('keyup', {
+        keyCode: 65
+      });
+    });
+    console.log(message.debug());
```

Comment:
Hello @Philipotieno ,
Please remove the console statement

Reply:
Done